contrib/win32/install/openssh.wixproj

```diff
@@ -14,6 +14,7 @@
             ProductVersion=$(ProductVersion);
         </DefineConstants>
         <DefineSolutionProperties>false</DefineSolutionProperties>
+        <!-- This is sufficient for x86 and x64 but gets overwritten below for ARM64 -->
```

Comment:
Maybe just reword this comment to clarify? Something like "The Target Path for x86 and x64 is located where Wix 3.11 is pre-installed on our build image. That value is overwritten for ARM64 so that it uses the TargetPath in the Win 3.14 directory that we install at the build runtime".